setup.py

```diff
@@ -244,6 +244,8 @@ def _check_ver(pyver, op, wanted):
     wanted = distutils.version.LooseVersion(wanted)
     if not isinstance(pyver, str):
         pyver = str(pyver)
+    if not isinstance(wanted, str):
+        wanted = str(wanted)
```

Comment:
What's this do?

Reply:
Prevents this on macOS

https://github.com/saltstack/salt/pull/59912/commits/6030613d66859a592419e70660b2080ef79230e2